indico/modules/rb_new/client/js/components/TimeRangePicker.jsx

```diff
@@ -0,0 +1,203 @@
+/* This file is part of Indico.
+ * Copyright (C) 2002 - 2018 European Organization for Nuclear Research (CERN).
+ *
+ * Indico is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Indico is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Indico; if not, see <http://www.gnu.org/licenses/>.
+ */
+
+import moment from 'moment';
+import React from 'react';
+import propTypes from 'prop-types';
+import {Dropdown} from 'semantic-ui-react';
+import {toMoment} from '../util';
+
+import './TimeRangePicker.module.scss';
+
+
+const _serializeTime = time => (time ? time.format('HH:mm') : null);
+
+function _humanizeDuration(duration) {
+    const hours = duration.hours();
+    const minutes = duration.minutes();
+    if (hours === 1 && minutes === 0) {
+        return `${hours} hour`;
+    } else if (hours !== 0) {
+        const fraction = minutes / 60;
+
+        return `${hours + fraction} hours`;
```

Comment:
You can limit it to only one line ``return `${hours + (minutes / 60)} hours`;``

Reply:
Are you sure I need parentheses here?

Reply:
actually you don't thanks to the operator precedence but I think it is usually clearer that way (at least to me)

Reply:
yeah, in such a case I prefer parentheses or not putting space around `/` if eslint allows that (PEP8 does, not sure about eslint)